less/pages/community.less

```diff
@@ -30,12 +30,20 @@
 }
 
 .communities-info {
-  display: flex;
```

Comment:
It's inline-block, which also works.

This way it just automatically goes to the next line if the width becomes too small. Then `text-align: center;` centers any lone elements, when they become lone (also works if two are side by side).

flex is really good if the sizes of things are fairly dynamic, but this probably doesn't needs it, and this way, we don't need to use media queries for updating flex.

In this particular case we would have 4 media queries if we wanted to be thorough, didn't seem necessary to me.